### ISSUE TITLE
Sirigne config

### ISSUE THREAD

**Issue by wpsimon09** [FELIXroboticsBV/Cura#9]
One of the requirements was to print message to the user when the amount of filament needed to print would not fit inside the nozzle. User should be informed about this. 

This can be achieved by exposing some form of syring configuration where users can select height and inner  radius of the syringe. Then I can calculate its volume and compare it to the estimated volume.

**Comment by wpsimon09** [comment]
This is partly done since you can select nozzle but not the syringe perse. Thus there is no way to calculate if the syringe will overflow or not.
